src/tokens-studio/spectrum2-colors/$themes.json

```diff
@@ -36,6 +37,12 @@
       "Icon.red": "1f734c4352da33631259a822a4377c36d9129856",
       "Icon.yellow": "34ea0effbfa548c92ac7a28428625b1fc9dbc47e",
       "Alias.overlay": "fa3f19d79b2633f5b0913c442831b94c56bfbd66",
+      "Alias.🚫 drop-shadow": "0e6900e9ce8ddf2c72e312dcc202679dd76f2516",
+      "Alias.drop-shadow.100": "439d299ee483470b504ec8b21ac2309f82da7849",
+      "Alias.drop-shadow.200": "6a04bd46ee13367ed8477c79bb1be3c4e5149748",
+      "Alias.drop-shadow.emphasized-default": "6845015b6692e43f3375947e20d74951d662ac28",
+      "Alias.drop-shadow.emphasized-hover": "e080d6090b53c910bca628a47dda47630c8ba759",
+      "Alias.drop-shadow.elevated": "7ffc61455bb815b6c1129d3d6ccfa224245fbfc0",
```

Comment:
I didn't modify these, are these due to other updates that I pulled into my branch? @lynnhao

Reply:
I'm not sure, these updates were [merged](https://github.com/adobe/spectrum-tokens-studio-data/pull/84) three weeks ago 🤔

Reply:
@karstens @nabuhasan yeah good point. I'm fine with either way. In the S2/Desktop file when we deprecate a component we usually do include a space, but whatever works best for tokens works for me!

Reply:
@lynnhao @nabuhasan I am also fine with either way, but I think, *with* space is a little bit more readable in the Figma UI. Lets go *with* space. 👍